### DISCUSSION TITLE
[574720]Little Big Workshop

### DISCUSSION THREAD

**Discussion by Xliang-Q** [pjy612/SteamManifestCache#153] [Share]
`appid:574720 `
keys:
```
574721;3639fec2e6886f768df5a18f61f57086cb4ac9eb28f086648d0db45ef6c81d56
```
[574721_4278201335399838254.manifest.txt](https://github.com/pjy612/SteamManifestCache/files/13761864/574721_4278201335399838254.manifest.txt)
email:xl453096651@126.com
code:3D7B94F2E8C1FA55

**Comment by pjy612** [comment]
查收

**Reply by Xliang-Q** [reply]
怎么避免CODE变动呀，台机有线连接，但是CODE还是变了  code:CB0460AD17568E78

**Reply by pjy612** [reply]
关 WLAN 里面的 随机硬件地址，有线和wifi 可能会 code 不一样
